BookList.js

```diff
@@ -0,0 +1,39 @@
+export default class BookList {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook = (bookTitle, bookAuthor) => {
+    const book = {
+      title: bookTitle,
+      author: bookAuthor,
+      id: this.generateId(),
+    };
+    this.books.push(book);
+  };
```

Comment:
- [ ] good job addressing the suggested changes 👍  however, right now if the user creates a new book it does get saved in the local storage but it doesn't get displayed in the browser.